src/sortingtraining/utils/ComparingUtil.java

```diff
@@ -16,17 +20,18 @@
 
     public void compare(int[] numbers) {
 
-        List<TimeMeasurement> times = new ArrayList<>();
-
-        times.add(new TimeMeasurement(TimerUtil.measure(new InsertionSort(), numbers), "Insertion Sort"));
-
-        times.add(new TimeMeasurement(TimerUtil.measure(new BubbleSort(), numbers), "Bubble Sort"));
-
-        times.add(new TimeMeasurement(TimerUtil.measure(new QuickSort(), numbers), "Quick Sort"));
-
-        times.add(new TimeMeasurement(TimerUtil.measure(new BucketSort(), numbers), "Bucket Sort"));
-
-        times.add(new TimeMeasurement(TimerUtil.measure(new DefaultSort(), numbers), "Default Library Sort"));
+        List<TimeMeasurement> times = Arrays.asList(
+                new TimeMeasurement(TimerUtil.measure(new InsertionSort(), numbers), "Insertion Sort"),
+                new TimeMeasurement(TimerUtil.measure(new ShellSort(), numbers), "Shell Sort"),
+                new TimeMeasurement(TimerUtil.measure(new BubbleSort(), numbers), "Bubble Sort"),
+                new TimeMeasurement(TimerUtil.measure(new SelectionSort(), numbers), "Selection Sort"),
+                new TimeMeasurement(TimerUtil.measure(new MergeSort(), numbers), "Merge Sort"),
+                new TimeMeasurement(TimerUtil.measure(new QuickSort(), numbers), "Quick Sort"),
+                new TimeMeasurement(TimerUtil.measure(new BucketSort(), numbers), "Bucket Sort"),
```

Comment:
Did you consider adding to enum another variable: String, with human readable Sorting alghoritm name?

Reply:
Great idea! Now when i know how to use streams i was also able to refactor this boilerplate code with stream! Check it out: 1633d00